src/api/resources/rest/sample.ts

```diff
@@ -57,18 +57,12 @@ export class SampleResource extends RESTResource<types.Sample> implements Permis
      * @param sampleId Sample id
      * @param collectionIds Array of collection ids
      */
-    public addToCollections(sampleId: number, collectionIds: number[]): Rx.Observable<void> {
-        return this.callMethod<void>(sampleId, 'add_to_collection', { ids: collectionIds });
-    }
-
-    /**
-     * Remove sample from collections.
-     *
-     * @param sampleId Sample id
-     * @param collectionIds Array of collection ids
-     */
-    public removeFromCollections(sampleId: number, collectionIds: number[]): Rx.Observable<void> {
```

Comment:
It's not possible to remove a sample from a collection from now on?

Reply:
I couldn't find a way that works.
If @JureZmrzlikar adds it to ReSDK, I can include it back